src/Auth/issuer.js

```diff
@@ -0,0 +1,19 @@
+import memoize from 'p-memoize';
+import { info } from 'heroku-logger';
+import { Issuer } from 'openid-client';
+
+/**
+ * Get the public key from the given issuer, via the service's
+ * OpenID Connect Discovery endpoint. <https://git.io/JfzJo>
+ *
+ * @param {String} url
+ * @returns {String}
+ */
+export default memoize(async url => {
+  info('Loading public key from Northstar.', { url });
+
+  const issuer = await Issuer.discover(url);
+  const keystore = await issuer.keystore();
+
+  return keystore.get({ kty: 'RSA' }).toPEM();
```

Comment:
Just confirming this is supposed to be `kty` and it is not a typo of `key` right? 😅

Reply:
Yeah you'd think so right!? 😂 It stands for ["key type"](https://tools.ietf.org/html/rfc7517#section-4.1).